rclcpp/include/rclcpp/logger.hpp

```diff
@@ -161,6 +162,16 @@ class Logger
   RCLCPP_PUBLIC
   void
   set_level(Level level);
+
+  /// Get the current logging directory.
+  /**
+   * For more details of how the logging directory is determined,
+   * see \ref rcl_logging_get_logging_directory.
```

Comment:
Also, it would be nice to document the return value and possible exceptions thrown.

Reply:
See https://github.com/ros2/rclcpp/pull/1509/commits/3c42fa6c3485a7cdfd72969b7757c0a29c1b065d.